### ISSUE TITLE
Try using tags for distribution gallery pages

### ISSUE THREAD

**Issue by OriolAbril** [arviz-devs/preliz#476]
I think it would be nice to have tags for the distribution gallery pages with general info like: continuous/discrete, bounded/unbounded, degrees of freedom, symmetry...

I will try using this extension: https://github.com/melissawm/sphinx-tags and see how it goes. As this is sphinx and it is a static site, fancy filtering won't be really possible, but in addition to having all the tags on the page, each tag should have its own page so people can see all distributions that are symmetric on the same page for example.

**Comment by aloctavodia** [comment]
What about this one? @CodingWithError

**Comment by CodingWithError** [comment]
yeah i would love to. Can you explain like do you want me to put the tags in READ ME ?

**Comment by aloctavodia** [comment]
Tag should appear in the Gallery https://preliz.readthedocs.io/en/latest/gallery_content.html. Try using this extension: https://github.com/melissawm/sphinx-tags

**Comment by aleicazatti** [comment]
Hi @CodingWithError,
Are you currently working on this issue? If not, I'd love to take it on. Let me know!
